_po/ja/how-to/laravel.po

```diff
@@ -193,38 +264,57 @@ msgid ""
 "            return static::query();\n"
 "        }"
 msgstr ""
+"    public static function search($keyword)\n"
+"    {\n"
+"        if(empty($keyword)){\n"
+"            return static::query();\n"
+"        }"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.